model/errors/rpc_error.go

```diff
@@ -98,6 +98,15 @@ var (
 	ErrInvalidAccountID = errors.New("invalid account id")
 )
 
+// auth service error
+var (
+	ErrInvalidRegNonce           = errors.New("invalid regNonce")
```

Comment:
add error description?

Reply:
Good catch. They are actually not used. I deleted all of them.